bin/fenix-qr-codes.js

```diff
@@ -31,7 +31,7 @@ if (!fs.existsSync(destDir)) {
         `utm_content=${id}`,
         `utm_medium=referral&utm_source=addons.mozilla.org`,
       ].join('&')
-      return QRCode.toFile(filePath, content);
+      return QRCode.toFile(filePath, content, {margin: 1});
```

Comment:
Why is this needed?

Reply:
I was just trying to match the image from the spec as closely as possible. Without setting `margin: 1` we end up with a margin that is quite a bit wider than the image in the spec, creating a bunch of extra whitespace inside the border.

Reply:
Because this PR doesn't include any code to actually display the component, let's punt on this for now, and we can make changes to the appearance of the panel in the follow-up PR.